### DISCUSSION TITLE
VS2022 VSIX project templates .Net 5.0?

### DISCUSSION THREAD

**Discussion by habaneando** [VsixCommunity/Community.VisualStudio.Toolkit#235] [Q&A]
Currently VS2022 is still using Framework 4.8 for VSIX project templates. 
I would like to know if Microsoft plans to upgrade them to 5.0. 
I will appreciate your support. Thanks a lot in advance.

**Comment by StevenRasmussen** [comment]
I'm not sure if you're aware but this is a community project with no direct relation to Microsoft and so you would need to post the question to an official Microsoft support area.  My opinion is that they will eventually support .Net 5.0+, perhaps in the next major release of Visual Studio (VS 2024?... seems to be about every 2 years) but that would be the earliest I would think.

**Reply by habaneando** [reply]
Hi Steven, 
I was trying to get the feeling from Mads Kristensen cuz it looks like he is leading this project [vsixcookbook](https://www.vsixcookbook.com/). 
Maybe I was wrong.  
Anyways I already posted my question here [Writing extensions just got easier by Mads](https://devblogs.microsoft.com/visualstudio/writing-extensions-just-got-easier/#comment-26280)
Thanks again for your reply.
